index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Terraria Bingo</title>
   </head>
-  <body style="margin: 0; height: 100%;">
+  <body style="margin: 0; height: 100%; font-family: 'andy';">
```

Comment:
@OfficialArms I wonder if we could have a style tag here to put all our default css things instead of inline css